core/store/src/migrations.rs

```diff
@@ -320,3 +326,126 @@ pub fn migrate_13_to_14(path: &String) {
 
     set_store_version(&store, 14);
 }
+
+pub fn migrate_14_to_15(path: &String) {
+    let store = create_store(path);
+    let trie_store = Box::new(TrieCachingStorage::new(store.clone(), TrieCache::new(), 0));
+    let trie = Trie::new(trie_store, 0);
+
+    let mut store_update = store.store_update();
+    let batch_size_limit = 10_000_000;
+    let mut batch_size = 0;
+
+    for (key, value) in store.iter_without_rc_logic(DBCol::ColOutcomesByBlockHash) {
+        let block_hash = CryptoHash::try_from_slice(&key).unwrap();
+        let block =
+            store.get_ser::<Block>(DBCol::ColBlock, &key).unwrap().expect("block should exist");
+
+        if block.chunks()[0].height_included() == block.header().height() {
+            let execution_outcome_ids = <HashSet<CryptoHash>>::try_from_slice(&value).unwrap();
+
+            let chunk = store
+                .get_ser::<ShardChunk>(DBCol::ColChunks, block.chunks()[0].chunk_hash().as_ref())
+                .unwrap()
+                .expect("chunk should exist");
+
+            let epoch_info = store
+                .get_ser::<EpochInfo>(DBCol::ColEpochInfo, block.header().epoch_id().as_ref())
+                .unwrap()
+                .expect("epoch id should exist");
+            let protocol_version = epoch_info.protocol_version;
+
+            let mut new_execution_outcome_ids = vec![];
+            let mut local_receipt_ids = vec![];
+            let mut local_receipt_congestion = false;
+
+            // Step 0: execution outcomes of transactions
```

Comment:
@khorolets FYI, this is probably the best resource to learn the order the execution inside a chunk is happening :smile: